templates/partial/_get-started-linux.html

```diff
@@ -44,9 +58,7 @@
             <div class="p-stepped-list__content">
               <div class="p-code-snippet">
                 <pre class="p-code-snippet__block"><code>microk8s enable dashboard</code></pre>
-                <pre class="p-code-snippet__block"><code>microk8s enable dns</code></pre>
```

Comment:
Is it necessary to remove enable dns?

Reply:
yes. it is already enabled by default, so this does nothing other than give the user a confusing error message